app/assets/javascripts/short-form/RentBurdenFileService.js.coffee

```diff
@@ -0,0 +1,86 @@
+RentBurdenFileService = ($http, $q, uuid, ListingDataService, ListingPreferenceService) ->
```

Comment:
love that you pulled this out into new file! 👏

Reply:
although I'm not sure, are we actually using this? Looks like there's special functions for rent burdened because there are multiple files being uploaded for the one preference, is that right?

I haven't seen it in the application before

Reply:
nevermind, just figured out how to get it to show up. need to report rent as >50% of income on Automated Test Listing